app/uk/gov/hmrc/nisp/views/includes/nirecordtaxyear.scala.html

```diff
@@ -86,7 +88,13 @@
                     classThreePayableWithPenalty <- niRecordTaxYear.classThreePayableByPenalty
                 ){
                         <p class="contributions-header">@Messages("nisp.nirecord.gap.youcanmakeupshortfall")</p>
-                        <p>@Html(Messages("nisp.nirecord.gap.payvoluntarycontrib",NispMoney.pounds(niRecordTaxYear.classThreePayable.getOrElse(0)),Dates.formatDate(classThreePayableWithPenalty.localDate),Dates.formatDate(classThreePayableByDate.localDate)))</p>
+
+                        @if(timePoint.toString < Dates.shortDate(classThreePayableByDate.localDate)) {
```

Comment:
Here I use a lexicographical comparison of two dates, as strings. As long as they are in the same format, then a string of "23-8-2019" is larger that "01-05-2019"